patches/runtime/0026-Patch-MS.Extensions.Logging-version.patch

```diff
@@ -0,0 +1,25 @@
+From 854f5fd743531c40f486305c9de3fe46e78eaabc Mon Sep 17 00:00:00 2001
+From: Chris Rummel <crummel@microsoft.com>
+Date: Mon, 2 Nov 2020 14:55:01 -0600
+Subject: [PATCH 26/26] Patch MS.Extensions.Logging version.
+
+---
+ .../wasm/debugger/BrowserDebugProxy/BrowserDebugProxy.csproj    | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/src/mono/wasm/debugger/BrowserDebugProxy/BrowserDebugProxy.csproj b/src/mono/wasm/debugger/BrowserDebugProxy/BrowserDebugProxy.csproj
+index ea99c465cbb..1658255f787 100644
+--- a/src/mono/wasm/debugger/BrowserDebugProxy/BrowserDebugProxy.csproj
++++ b/src/mono/wasm/debugger/BrowserDebugProxy/BrowserDebugProxy.csproj
+@@ -5,7 +5,7 @@
+   </PropertyGroup>
+ 
+   <ItemGroup>
+-    <PackageReference Include="Microsoft.Extensions.Logging" Version="3.1.2" />
++    <PackageReference Include="Microsoft.Extensions.Logging" Version="$(MicrosoftExtensionsLoggingPackageVersion)" />
```

Comment:
@crummel what is going to set the `MicrosoftExtensionsLoggingPackageVersion` variable?

Reply:
This should be getting set in SBRP or previously-source-built packages, and Arcade should import then before this gets evaluated.